src/server.c

```diff
@@ -5389,14 +5393,18 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "client_recent_max_output_buffer:%zu\r\n"
             "blocked_clients:%d\r\n"
             "tracking_clients:%d\r\n"
-            "clients_in_timeout_table:%llu\r\n",
+            "clients_in_timeout_table:%llu\r\n"
+            "tot_blocking_keys:%lu\r\n"
+            "tot_blocking_keys_on_nokey:%lu\r\n",
```

Comment:
```suggestion
            "total_blocking_keys:%lu\r\n"
            "total_blocking_keys_on_nokey:%lu\r\n",
```
For consistency with other usages of "total"

Reply:
maybe also replace `on_nokey` with another term? maybe `on_del`?
or `_removed`?

Reply:
soloestoy [suggested](https://github.com/redis/redis/pull/11012#discussion_r1030144253) 
> `unblock_on_key_changed` or `unblock_on_nokey_or_wrongtype`?

i'd argue that when a key is overwritten with another type, we can view it as "removal" (and re-insertion).

Reply:
@oranagra unblock on nokey can also happen in cases the key type was changed. so I just followed the same semantics presented in #11310 - I can take the advice from @soloestoy:
total_blocking_keys_on_nokey_or_wrongtype (great simple name :))

Reply:
then i guess we can stick with `nokey`, it's enough that it's documented what it means, and it's easy to grep for it (i.e. we define a short term, and document what it means).

IMO, just change `tot` to `total`

Reply:
done